raspimouse/src/raspimouse_component.cpp

```diff
@@ -33,9 +33,9 @@ constexpr auto use_light_sensors_param = "use_light_sensors";
 constexpr auto odometry_scale_left_wheel_param = "odometry_scale_left_wheel";
 constexpr auto odometry_scale_right_wheel_param = "odometry_scale_right_wheel";
 
-constexpr auto wheel_diameter = 0.048;
-constexpr auto wheel_base = 0.09;
-constexpr auto PULSES_PER_REVOLUTION = 400.0;
+constexpr auto wheel_diameter_param = "wheel_diameter";
+constexpr auto wheel_base_param = "wheel_base";
+constexpr auto PULSES_PER_REVOLUTION_PARAM= "PULSES_PER_REVOLUTION";
```

Comment:
ROSパラメータ名は小文字に、ソースコード内の定数は大文字で表記して下さい。

```suggestion
constexpr auto WHEEL_DIAMETER_PARAM = "wheel_diameter";
constexpr auto WHEEL_BASE_PARAM = "wheel_base";
constexpr auto PULSES_PER_REVOLUTION_PARAM= "pulses_per_revolution";
```

Reply:
追加でほかの変数名も変更しますか？  
https://github.com/rt-net/raspimouse2/blob/ea91d066b4fc2eb3941e7489c8343acf640f1c53/raspimouse/src/raspimouse_component.cpp#L31-L34

Reply:
これらについては、今回は変更不要です。

リファクタリングのみのPRで対応するか、https://github.com/rt-net/raspimouse2/pull/36#discussion_r751816928
の変更を実施するときについでに作業しましょう。